go/vt/mysqlctl/backup.go

```diff
@@ -134,9 +149,14 @@ func Backup(ctx context.Context, params BackupParams) error {
 	if err != nil {
 		return vterrors.Wrap(err, "failed to find backup engine")
 	}
-
+	// Scope stats to selected backup engine.
+	beParams := params.Copy()
+	beParams.Stats = params.Stats.Scope(
+		stats.Component(stats.BackupEngine),
+		stats.Implementation(backupEngineImplementation),
+	)
 	// Take the backup, and either AbortBackup or EndBackup.
-	usable, err := be.ExecuteBackup(ctx, params, bh)
+	usable, err := be.ExecuteBackup(ctx, beParams, bh)
```

Comment:
why copy , why not pass reference of &param

Reply:
The `stats` object passed to `be.ExecuteBackup` has a different scope than the one passed in to this function.